com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/Editor/ShaderGraph/Targets/BuiltInTarget.cs

```diff
@@ -81,6 +81,10 @@ sealed class BuiltInTarget : Target, IHasMetadata
         List<string> m_SubTargetNames;
         int activeSubTargetIndex => m_SubTargets.IndexOf(m_ActiveSubTarget);
 
+        // Subtarget Data
+        [SerializeField]
+        List<JsonData<JsonObject>> m_SubTargetData = new List<JsonData<JsonObject>>();
```

Comment:
I think I know why you used JsonObject here, but for clarity - this was done because you cant do JsonData<SubTarget>, correct?

Reply:
Yeah, I needed the most generic implementation possible, so it's compatible with the already existing sub target data in HDRP.

Reply:
What are the possible types that go in this list?